src/program/types/AwaitExpression.js

```diff
@@ -0,0 +1,20 @@
+import Node from '../Node.js';
+
+const noop = () => {};
+
+export default class AwaitExpression extends Node {
+	static removeAsync(code, transforms, async, start, callback = noop) {
+		if (transforms.asyncAwait && async) {
+			code.remove(start, start + 6);
```

Comment:
Magic number 6 is mysterious here. Better as named variable, or with a comment?

Reply:
could this be "async ".length?